asciidoc/courses/aura-dashboards/modules/1-getting-started/lessons/1-what-are-dashboards/questions/1-purpose.adoc

```diff
@@ -1,11 +1,11 @@
 [.question]
-= Purpose of Aura Dashboards
+= Purpose of Neo4j Dashboards
 
-Which of the following statements best describes the purpose of Aura's integrated dashboard tool?
+Which of the following statements best describes the purpose of Neo4j's integrated dashboard tool?
 
-* [x] The integrated dashboard tool aims to provide a visual interface for exploring data insights directly within the Neo4j Aura environment without having to write Cypher queries.
+* [x] The integrated dashboard tool provides a visual interface for exploring data insights directly within Neo4j without having to write Cypher queries.
```

Comment:
Is this correct? You still have to write Cypher queries to get the most out of the dashboards.